pkg/autoops/command/webhook.go

```diff
@@ -119,7 +119,9 @@ func (h *webhookCommandHandler) ChangeWebhookName(ctx context.Context, cmd *auto
 	return h.publisher.Publish(ctx, event)
 }
 
-func (h *webhookCommandHandler) ChangeWebhookDescription(ctx context.Context, cmd *autoopspb.ChangeWebhookDescriptionCommand) error {
+func (h *webhookCommandHandler) ChangeWebhookDescription(
+	ctx context.Context,
+	cmd *autoopspb.ChangeWebhookDescriptionCommand) error {
```

Comment:
Could you please refactor it as follows as we do in other files to keep consistency?

E.g.

```go
func (h *webhookCommandHandler) ChangeWebhookDescription(
    ctx context.Context,
    cmd *autoopspb.ChangeWebhookDescriptionCommand,
) error {
    ....
}
```

Reply:
I fixed it, thank you.